deepsea.spec

```diff
@@ -32,6 +32,9 @@ BuildRequires:  salt-master
 Requires:       salt-master
 Requires:       salt-minion
 Requires:       python-ipaddress
+%if 0%{?leap_version} == 420200
```

Comment:
@rjfd btw I checked and in sle the salt-master brings in python-netaddr as a dependency
so we should keep an eye on that when sle updates the salt-master.